rpcs/ledgerService.go

```diff
@@ -229,10 +230,37 @@ func (ls *LedgerService) ListenForCatchupReq(reqs <-chan network.IncomingMessage
 // a blocking function for handling a catchup request
 func (ls *LedgerService) handleCatchupReq(ctx context.Context, reqMsg network.IncomingMessage) {
 	var res WsGetBlockOut
+	target := reqMsg.Sender.(network.UnicastPeer)
+	var respTopics network.Topics
+
+	if target.Version() == "2.1" {
```

Comment:
I think that we want to flip this around. If we have version "1", then we need to use the old implementation, otherwise, we should use the new implementation.